### DISCUSSION TITLE
Clear blank screen lines after cursor prompt

### DISCUSSION THREAD

**Discussion by resolritter** [kovidgoyal/kitty#3569] [Q&A] [answered]
I would like to remove blank lines after the command prompt (`>`). As shown in the following video (and made more explicit by the mouse cursor's movement), when I move around the terminals in the tiling window manager grid, I end up with some blank space at the bottom.

https://user-images.githubusercontent.com/17429390/116953939-eeeabb80-ac64-11eb-8813-3221bf3ff700.mp4

I have tried searching for some escape sequence to achieve this, but could not find it.

Thanks in advance!

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf.html#opt-kitty.scrollback_fill_enlarged_window

**Reply by resolritter** [reply]
Great! More explicitly, I was looking for `scrollback_fill_enlarged_window yes` (pretty new option, [0.20](https://github.com/kovidgoyal/kitty/blob/49459b3774c461cafe0028059e2bf3dac94fc979/docs/changelog.rst#0200-2021-04-19)).
